src/patterns/dosdp-patterns/boneLengthAlongAxis.yaml

```diff
@@ -0,0 +1,74 @@
+pattern_name: boneLengthAlongAxis
+
+pattern_iri: http://purl.obolibrary.org/obo/oba/patterns/boneLengthAlongAxis.yaml
+
+description: "A bone length trait that is the line along an axis of a bone."
+
+#  examples:
+#    - example_IRI-1  # term name
+#    - example_IRI-2  # term name
+#    - example_IRI-3  # term name
+
+contributors:
+  - https://orcid.org/0000-0002-7073-9172  # David Osumi-Sutherland
+  - https://orcid.org/0000-0001-8314-2140  # Ray Stefancsik
+  - https://orcid.org/0000-0002-7356-1779  # Nicolas Matentzoglu
+  - https://orcid.org/0000-0003-2699-3066  # Meghan A. Balk
+
+classes:
+  skeletal_element: UBERON:0004765
+  anatomical_entity: UBERON:0001062
+  length: PATO:0000122
+  line: BFO:0000026  # one-dimensional spatial region
+  physical_object_quality: PATO:0001241
+  anatomical_axis: 'BSPO:0000010'
+
+relations:
+  characteristic_of: RO:0000052
+  follows_axis: 'BSPO:0015002'
+
+annotationProperties:
+  exact_synonym: oio:hasExactSynonym
+  related_synonym: oio:hasRelatedSynonym
+  xref: oio:hasDbXref  #  measured_in_taxon #  not_measured_in_tax
+
+vars:
+  bone: 'skeletal element'
+  axis: 'anatomical_axis'
+  fovt: "'physical_object_quality'"  # FOVT id
+
+name:
+  text: "%s length"
+  vars:
+    - bone
+    - axis
+
+def:
+  text: "The length of a %s along the %s"
+  vars:
```

Comment:
just realized it should be "The length of a %s along the %s axis"

Reply:
I have updated the text.